test/features/resource_availability.feature

```diff
@@ -1,252 +1,113 @@
 @resource_visibility
 Feature: Re-identification risk governance acknowledgement or Resource visibility
 
-
-    Scenario: Sysadmin creates dataset with Contains de-identified data is YES and Resource visibility is TRUE and Acknowledgement is NO
-        Given "SysAdmin" as the persona
-        When I log in
-        And I go to "/dataset/new"
-        Then I fill in "title" with "Contains de-identified data YES visibility TRUE acknowledgment NO" 
-        Then I fill in "notes" with "hide resource"
-        Then I execute the script "document.getElementById('field-organizations').value=jQuery('#field-organizations option').filter(function () { return $(this).html() == 'Test Organisation'; }).attr('value')"
-        Then I select "False" from "private"
-        Then I fill in "version" with "1"
-        Then I fill in "author_email" with "test@test.com"
-        Then I select "YES" from "de_identified_data"
-        And I press the element with xpath "//form[contains(@class, 'dataset-form')]//button[contains(@class, 'btn-primary')]"
-        And I wait for 10 seconds
-        Then I execute the script "document.getElementById('field-image-url').value='http://ckanext-data-qld.docker.amazee.io/'"
-        Then I fill in "name" with "Resource hidden"
-        Then I fill in "description" with "hide resource"
-        Then I select "TRUE" from "resource_visible"
-        Then I select "NO" from "governance_acknowledgement"
-        Then I press the element with xpath "//button[@value='go-metadata']"
-        And I wait for 10 seconds
-        Then I should see "Data and Resources"
-
-
-    Scenario: Sysadmin creates dataset with Contains de-identified data is NO and Resource visibility is TRUE and Acknowledgement is NO
+    Scenario: "Create resource_availability test data"
```

Comment:
Scenario names should describe the action taken (possibly including the role taking it) and the expected results.

Reply:
Updated scenario name